server/src/main/java/org/elasticsearch/cluster/DiffableUtils.java

```diff
@@ -148,6 +149,15 @@ private DiffableUtils() {
         return new ImmutableOpenMapDiff<>(in, keySerializer, new DiffableValueReader<>(reader, diffReader));
     }
 
+    /**
+     * Returns an empty diff for the given key and value serializers
+     */
+    public static <K, T extends Diffable<T>> MapDiff<K, T, ImmutableOpenMap<K, T>> emptyOpenMapDiff(KeySerializer<K> keySerializer,
```

Comment:
I don't think this change is needed in the file? Because the new method isn't used?

Reply:
Yep, this is a leftover I forgot to remove. I'll remove it!